src/main.cpp

```diff
@@ -5838,6 +5838,9 @@ CMutableTransaction CreateNewContextualCMutableTransaction(const Consensus::Para
         mtx.nVersionGroupId = OVERWINTER_VERSION_GROUP_ID;
         mtx.nVersion = 3;
         // Expiry height is not set. Only fields required for a parser to treat as a valid Overwinter V3 tx.
+
+        // TODO: In future, when moving from Overwinter to Sapling, it will be useful
+        // for the expiry height to be set to min(blocks_until_upgrade, default_expiry_height)
```

Comment:
`min(activation_height - 1, default_expiry_height)` (since `nExpiryHeight` is the last mineable height).